airbyte-commons-protocol/src/main/java/io/airbyte/commons/protocol/migrations/AirbyteMessageMigrationV1.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.commons.protocol.migrations;
+
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.version.Version;
+import io.airbyte.protocol.models.v0.AirbyteMessage;
+import jakarta.inject.Singleton;
+
+/**
+ * Placeholder AirbyteMessage Migration from v0 to v1
+ */
+@Singleton
+public class AirbyteMessageMigrationV1 implements AirbyteMessageMigration<AirbyteMessage, io.airbyte.protocol.models.AirbyteMessage> {
+
+  @Override
+  public AirbyteMessage downgrade(io.airbyte.protocol.models.AirbyteMessage message) {
+    return Jsons.object(Jsons.jsonNode(message), AirbyteMessage.class);
+  }
+
+  @Override
+  public io.airbyte.protocol.models.AirbyteMessage upgrade(AirbyteMessage message) {
+    return Jsons.object(Jsons.jsonNode(message), io.airbyte.protocol.models.AirbyteMessage.class);
+  }
+
+  @Override
+  public Version getPreviousVersion() {
+    return new Version("0.0.0");
+  }
+
+  @Override
+  public Version getCurrentVersion() {
+    return new Version("1.0.0");
```

Comment:
Does it make sense to define these versions somewhere in a constants class?  Something like PREVIOUS and CURRENT, as well as specific version numbers so that when we want to change, we can just basically update that class (e.g. point CURRENT to a newly added VERSION_X constant).  Feels like otherwise we are going to have a few versions of these strings floating around.  Not sure exactly how to reduce the duplication, but it feels like centralizing these would help.

Reply:
Those values are tied to migration itself. Currently, we can add V2 that would have a previous v1, current v2.

We currently have https://github.com/airbytehq/airbyte/blob/4974855b415ea4812e0d6647efb367ad837d8733/airbyte-commons/src/main/java/io/airbyte/commons/version/AirbyteProtocolVersion.java#L7, I could add some constants `V0`, `V1` in there.

Reply:
Updated